README.md

```diff
@@ -1110,6 +1110,12 @@ IEEE International Conference on Multisensor Fusion and Integration for Intellig
 *[X-maps: Direct Depth Lookup for Event-based Structured Light Systems](https://openaccess.thecvf.com/content/CVPR2023W/EventVision/papers/Morgenstern_X-Maps_Direct_Depth_Lookup_for_Event-Based_Structured_Light_Systems_CVPRW_2023_paper.pdf)*,  
 IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2023. [Project page](https://fraunhoferhhi.github.io/X-maps/), [Code](https://github.com/fraunhoferhhi/X-maps).  
 
+<a name="voxel-mono"></a>
+### Monocular Voxel Reconstruction
+- <a name="Chen23icvr"></a>Chen, H., Chung, V., Tan, L., Chen, X.,  
+*[Dense Voxel 3D Reconstruction Using a Monocular Event Camera](https://doi.org/10.1109/ICVR57957.2023.10169359)*,  
+International Conference on Virtual Reality (ICVR), 2023. [IEEEXplore](https://ieeexplore.ieee.org/document/10169359), [Arxiv](https://arxiv.org/abs/2309.00385), [Dataset](https://www.kaggle.com/datasets/hche8927/synthevox3d).
```

Comment:
Int. Conf. Virtual Reality
No need for the IEEEXplorer link. The DOI link will take you there.
[Arxiv ] -> change to [PDF] , replacing the "abs" part in the link with "pdf".

Reply:
[620ac5c](https://github.com/uzh-rpg/event-based_vision_resources/pull/235/commits/620ac5ca12c38ed4e9327d86f183f907716c9aef)
Changed "International Conference on Virtual Reality" to "Int. Conf. Virtual Reality". 
Removed the IEEEXplore link and changed the ArXiv link to a PDF link (/abs/ to /pdf/).